### ISSUE TITLE
User accessibility

### ISSUE THREAD

**Issue by edmondchuc** [CSIRO-enviro-informatics/policies-catalogue#4]
Fix user accessibility issues like missing labels and text. Ensure that the site passes the WAVE Evaluation Tool Chrome extension.

**Comment by edmondchuc** [comment]
https://github.com/CSIRO-enviro-informatics/policies-catalogue/commit/2c713066d1deb9f9ce44e2cbc2591ff3815c673f
https://github.com/CSIRO-enviro-informatics/policies-catalogue/commit/5b5a63ab653510b547616538f337a1716f09592a
